src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,40 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
```

Comment:
As I understand, it's better to add a service layer to avoid directly interacting with the DAO in the main class. Instead, interactions should occur through the service layer. Maybe it`s not necessary for this task.

Reply:
I think it is not necessary in this task